Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 6.0
+// swift-tools-version: 5.10
```

Comment:
Shouldnt this rather be 5.9 to have largest compatibility with Xcode 15 versions? Specially given minimum supported Xcode version by brew is Xcode 15.0 which is having Swift 5.9, while Swift 5.10 is reaching only Xcode versions >= 15.3

Reply:
Raised https://github.com/a7ex/xcresultparser/pull/43 to address that.